### ISSUE TITLE
BaseVocabularyView and AJAX select widget does not work for vocabularies with non-textual values

### ISSUE THREAD

**Issue by datakurre** [plone/plone.app.content#144]
Currently this seems to be a feature, because of possible issues in our original Select2 widget integration:

https://github.com/plone/plone.app.content/commit/9927a5251bcf3b7a9262ee85a097d45d32181300

**Comment by thet** [comment]
@datakurre  I think the ``@@getVocabulary`` view should be fixed to support values again while avoiding the unicode decode errors i encountered back then - as discussed in the commit comments which you referenced.

**Comment by datakurre** [comment]
@thet I agree. It looked like a rabbit hole, so just now I could only open an issue.
